install.sh

```diff
@@ -3342,23 +3395,25 @@ execute_installation() {
     echo # For visual separation
 
     # Automatic updates
-    if setup_automatic_update_checks; then
+    if ! is_update_only && setup_automatic_update_checks; then
         echo # For visual separation
     fi
 
-    # Install the hooks into existing local repositories
     if ! should_skip_install_into_existing_repositories; then
         if is_single_repo_install; then
-            install_hooks_into_repo "$(pwd)" || return 1
+            install_hooks_into_repo "$(pwd)"
         else
-            install_into_existing_repositories
+            if ! is_update_only; then
```

Comment:
Nit:
```
if is_update_only; then
  install_into_registered_repositories
else
  install_into_existing_repositories
fi
```

Reply:
Its ok as it is IMO. We want both to happen when we are not int update mode (user requested an update, not triggered from the hooks)